### ISSUE TITLE
Better support/way to map leader to commands

### ISSUE THREAD

**Issue by anhdle14** [asvetliakov/vscode-neovim#550]
I want to map:

```vim
let mapleader="\<space>"
nnoremap <leader>w <c-w>
```

But this won't work out of the box from `init.vim`. Also here are the things that I tried.

1. `keybindings.json`. This work great until I try `R`, I thought it should go to insert mode, but it is not (i.e. `(Space) was pressed. Waiting for second key of chord`)
  ```json
   {
       "command": "vscode-neovim.send",
       "key": "space w",
       "when": "editorTextFocus && neovim.init && neovim.mode != insert",
       "args": "<C-W>"
   }
  ```
2. I tried various way to some how send <C-W> with VSCodeCall/Notify functions but to no prevail.
3. Re-mapping all <C-W> within the vscode-neovim does the trick but it is not maintainable.

So basically like the title, is there a way to map leader to commands natively from `init.vim` or not?

**Comment by theol0403** [comment]
I tried this out, and all you need to do is use `nmap` and not `nnoremap`. This is to make the mapping use the window commands in https://github.com/asvetliakov/vscode-neovim/blob/master/vim/vscode-window-commands.vim, and not the default neovim ones.

**Comment by priyamDalmia** [comment]
This might help. Put this in the `settings.json` file. The contraints under the 'when' param here should do the job. 

```
{
        "key": "space e",
        "command": "workbench.action.toggleSidebarVisibility",
        "when": "editorTextFocus && neovim.ctrlKeysNormal && neovim.init && neovim.mode != 'insert'"
},
```
